### PR TITLE
Update font-charter

### PR DESCRIPTION
**Pull request by gtmgianni**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

File: Casks/font-charter.rb [hunk 1]
```diff
@@ -2,12 +2,12 @@
   version :latest
   sha256 :no_check
 
-  url 'https://practicaltypography.com/fonts/charter.zip'
+  url 'https://practicaltypography.com/fonts/Charter%20200512.zip'
```
Comment (vitorgalvao) [inline Casks/font-charter.rb:5]: What does the `200512` stand for? Does that change between updates?

Comment (gtmgianni) [reply Casks/font-charter.rb:5]: I'm not sure. I haven't used this cask before – it was just flagged up by my script while checking for broken download URLs. Should I assume it's a version number and update the cask accordingly?

Comment (vitorgalvao) [reply Casks/font-charter.rb:5]: > Should I assume it's a version number and update the cask accordingly?

Yes, please. Just realised it probably stands for a release date—YYMMDD, 12<sup>th</sup> May 2020.

Comment (gtmgianni) [reply Casks/font-charter.rb:5]: Ah good point – I thought that at first (though I thought it was YYYYMMDD), but then realised the %20 represented a space character. Didn't consider a YYMMDD format though – I'll update the cask now.

**Pull request merged**